coinlib/bin/build_windows_crosscompile.dart

```diff
@@ -0,0 +1,45 @@
+import 'dart:io';
+import 'docker_util.dart';
+
+/// Build a Windows DLL for secp256k1 using a Dockerfile string.
+
+String dockerfile = r"""
+FROM debian:bullseye
+
+# Install dependenices.
+RUN apt-get update -y \
+  && apt-get install -y autoconf libtool build-essential git cmake
```

Comment:
Add `gcc-mingw-w64` as in
`  && apt-get install -y autoconf libtool build-essential git cmake gcc-mingw-w64`

Reply:
Thanks for that, I had to add further changes in https://github.com/peercoin/coinlib/pull/25/commits/a813cfbc037ec97bb44ac3b1bd68d066566213ea but I got it to provide an output now.